admin/app/helpers/solidus_admin/layout_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module SolidusAdmin
+  # Helpers for the admin layout
+  module LayoutHelper
+    # @param store_name [String]
+    # @param controller_name [String]
+    # @return [String] HTML title
+    def solidus_admin_title(store_name: current_store.name, controller_name: controller.controller_name)
+      "#{store_name} - #{t("solidus_admin.#{controller_name}")}"
```

Comment:
Having the title scoped to the controller is great, by using a relative key `t('.title')` rails will check for an action level translation first, giving even more granularity in case it's needed:

- `"en.solidus_admin.#{controller_name}.#{action_name}.title"`
- `"en.solidus_admin.#{controller_name}.title"`

See https://api.rubyonrails.org/classes/ActionView/Helpers/TranslationHelper.html#method-i-translate

Reply:
I'm afraid that doesn't work in our context, as the method is called from within a helper (at least, I didn't manage to make it work). In any case, I only wanted to leave an acceptable starting point and make the a11y happy. We probably need to define how we want our HTML titles to look, maybe in the context of breadcrumbs or not :slightly_smiling_face:

Reply:
Oh, that's right, it probably works only if generated in the controller or in the (non-layout) template.
This is fine for now